configure.cmd

```diff
@@ -140,48 +140,13 @@ REM Parse command line parameters
                 set VS_VERSION=%3
                 echo Visual Studio Environment set to !BUILD_ENVIRONMENT!!VS_VERSION!-!ARCH!
             )
-            if "!VS_VERSION!" == "10" (
```

Comment:
Does this also work for VS 2010?

Reply:
I see no reason it shouldn't. CMake supports the -A option for `VS_VERSION>=8`.  
That said, I don't have Visual Studio 2010 on hand to test.